.github/workflows/fuzzing.yml

```diff
@@ -0,0 +1,75 @@
+name: Fuzzing
+
+on:
+  schedule:
```

Comment:
Fuzzers run in 2 scenarios:
- On PR, where they'll run with `--fuzz-type local-regression` in which it'll download the corpus from Fuzzit and run them against the new code.
- On scheduled run, where new fuzzer binaries will be built from HEAD of the branch and submitted to Fuzzit with `--fuzz-type fuzzing` to run through the next day.

How can we reuse the steps listed here?